yellowbrick/features/pca.py

```diff
@@ -0,0 +1,103 @@
+##########################################################################
+## Imports
+##########################################################################
+import matplotlib.pyplot as plt
+
+from yellowbrick.features.base import DataVisualizer
+
+from sklearn.pipeline import Pipeline
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+##########################################################################
+##2D PCA Visualizer
+##########################################################################
+class PCA2D(DataVisualizer):
```

Comment:
This is _exactly_ what we hoped to see from new visualizers - good naming, good documentation/docstrings, tests; it's the whole package!